src/Nulls.jl

```diff
@@ -45,9 +45,6 @@ Base.next(::Null, ::Bool) = (null, true)
 Base.done(::Null, b::Bool) = b
 
 Base.promote_rule{T}(::Type{T}, ::Type{Null}) = Union{T, Null}
-Null(x) = null
-Base.convert(::Type{Null}, ::Null) = null
-Base.convert{T}(::Type{Union{T, Null}}, x) = convert(T, x)
```

Comment:
Can we keep this one? Otherwise, I see errors like
```julia
LoadError: MethodError: Cannot `convert` an object of type Float64 to an object of type Union{Int64, Nulls.Null}
This may have arisen from a call to the constructor Union{Int64, Nulls.Null}(...),
since type constructors fall back to convert methods.
```
when trying to convert a `Vector{?Float64}([1.0, 2.0])` to a `Vector{?Int}`